core/src/test/java/org/apache/calcite/test/TypeCoercionConverterTest.java

```diff
@@ -173,6 +173,20 @@ public static void checkActualAndReferenceFiles() {
     sql(sql).ok();
   }
 
+  @Test void testInsertValuesDifferentColumnTypes() {
```

Comment:
`testInsertValuesDifferentColumnTypes` may not be a perfect name here, how about `testInsertValuesWithConvertibleTypes` ?

Reply:
Thanks, fixed